pset-iterators.js

```diff
@@ -160,7 +186,12 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 6:')
 
 // Add your code below this line
+console.log(presidentsParties [0])
+const firstRepublican = presidentsParties.find(function(presidentParty){
+  return presidentParty.party === 'Republican'
+})
 
+console.log(firstRepublican)
```

Comment:
@ChicoMangusta Good job here reusing the variable `presidentsParties` to return the first occurrence of the republican party. 

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L175)